test/ui-test/src/screens/components/community_back_up_private_key_popup.py

```diff
@@ -0,0 +1,18 @@
+from drivers.SquishDriver import *
+
+from .base_popup import BasePopup
+
+
+class BackUpCommunityPrivateKey(BasePopup):
+    def __init__(self):
+        super(BackUpCommunityPrivateKey, self).__init__()
+        self._copy_private_key_button = Button('copyCommunityPrivateKey')
+        self._community_private_key_text_edit = TextEdit('transferOwnerShipTextEdit')
```

Comment:
regarding this one, it is tricky to import the enums from `StatusCommunityScreen.py` where it lives and i think there was a discussion between @noeliaSD and @StateOf-Vlado that it is accepted to keep it this way now (but i could be wrong :D )

Reply:
Yeah, we will be updating the code to create the ui objects used in a specific page inside the constructor, and then use this ui objects directly as object inside the class (more object-oriented pattern), so we will be able to replace the enums, since we will manage the ui components as objects in code, and the string to link from `name.py` to `Screens` will be ONLY used ONCE in the constructor.